WebDevStudios/nextjs-wordpress-starter
# Feature/lazy blocks

Closes N/A

### Link

Link to the branch preview generated by Vercel.

### Description

* Add support for lazy blocks MediaText and Hero blocks

### Screenshot

![ 2021-02-01 at 11 20 14 ](https://user-images.githubusercontent.com/12365909/106486270-774b9b80-647f-11eb-87e0-44e947b4deb2.png)

### Verification

How will a stakeholder test this?

1. Make sure the lazy blocks plugin is installed
2. Add lazyblocks/mediatext and/or lazyblocks/hero blocks to the page
3. Fill in the fields
4. Ensure the block displays properly on the front end
